packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1551,7 +1551,106 @@ in the request is deleted.
 
 Updated [User](#user-model)
 
-#### Update Profile
+#### Update Current User's Profile
+
+
+<ApiOperation method="post" url="/api/v1/users/me" /> <SupportsCors />
+
+Updates current user's profile with partial update semantics
+
+##### Request Parameters
+
+
+| Parameter     | Description                                                          | Param Type   | DataType                                    | Required |
+| :------------ | :------------------------------------------------------------------- | :----------- | :------------------------------------------ | :------- |
+| profile       | Updated profile for user                                             | Body         | [Profile Object](#profile-object)           | FALSE    |
+
+End user can only update `profile` with this request. To update credentials, use [Update Profile with ID](#update-profile-with-id).
+
+>**Note:** An end user can only update profile properties which the user has write access for. To update user permissions for a schema property,
+use [Update User Profile Schema Property](/docs/reference/api/schemas/#update-user-profile-schema-property)
+
+##### Response Parameters
+
+
+Updated [User](#user-model)
+
+##### Request Example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@update.example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
```

Comment:
Consider omitting one or two of these fields (e.g., omit firstName/lastName).  Currently the response includes all of these properties, so you are not illustrating the partial aspect of the update.

Reply:
I copied this piece from the update user with ID portion. Although this seems to be a good suggestion to me. @jakubvul what do you think? Do you want me to make the same change(omit some fields in request body to illustrate partial update) for the update user with ID section as well?

Reply:
Sounds good to me!